scalding-core/src/main/scala/com/twitter/scalding/typed/cascading_backend/AsyncFlowDefRunner.scala

```diff
@@ -274,16 +298,15 @@ class AsyncFlowDefRunner(mode: CascadingMode) extends Writer {
         dest.writeFrom(pipe)(fd, mode)
       }
 
-      def force[A](init: TypedPipe[A], opt: TypedPipe[A]): Unit = {
+      def force[A](init: TypedPipe[A], opt: TypedPipe[A], uuid: UUID): Unit = {
         val pipePromise = Promise[TypedPipe[A]]()
         val fut = pipePromise.future
         // This updates mutable state
         val sinkOpt = updateState { s =>
           val (nextState, added) = s.addForce(conf, init, opt, fut)
           if (added) {
-            val uuid = UUID.randomUUID
```

Comment:
I worry about this change in semantics. If we rerun the execution (say because we have withNewCache) will we correctly overwrite the data? Some taps fail on overwrite to protect data. Why do we want a UUID per Execution verses a new UUID each time we try to run?